File: python_pip_examples/websocket/exchange_websocket.py
from bitcoinaverage import ExchangeWebsocketClient

if __name__ == '__main__':
    secret_key = '' or input('Enter your secret key: ')
    public_key = '' or input('Enter your public key: ')

    print('Connecting to the exchange websocket...')
    ws = ExchangeWebsocketClient(public_key, secret_key)
    ws.exchange_data('bitstamp')

File: node_npm_examples/websocket/exchangeWebsocket.js
const ba = require('bitcoinaverage');

var publicKey = 'yourPublicKey';
var secretKey = 'yourSecretKey';
var ws = ba.websocketClient(publicKey, secretKey);

// Connecting to the Exchange websocket and printing Bitstamp data
ws.connectToExchangeWebsocket('bitstamp', function (response) {
    console.log(JSON.stringify(response, null, 4));
});
